### PR TITLE
Longest Increasing Subsequence #2253

### PR DESCRIPTION
**Pull request by biekymahato**
ISSUE NUMBER #2253 
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Languages Used:
- c#

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by biekymahato** [comment]
@asha15 @jainaman224  please have a look.

**Comment by somya-kapoor** [comment]
@biekymahato Please add the issue number.

**Comment by biekymahato** [comment]
@somya-kapoor Done.

**Comment by raghav-dalmia** [comment]
Your PR have 2 files. Remove second one. Also squash your commits.

File: Longest Increasing Subsequence(c#)/LIS.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+        //Initialize LIS values for all indexes
+        for ( i = 0; i < n; i++ )
+            lis[i] = 1;
+
+        //Compute optimized LIS values in bottom up manner
+        for ( i = 1; i < n; i++ )
+            for ( j = 0; j < i; j++ )
+                        if ( arr[i] > arr[j] && lis[i] < lis[j] + 1)
+                    lis[i] = lis[j] + 1;
```
Comment (asha15) [inline Longest Increasing Subsequence(c#)/LIS.cs]: correct the indentation

File: Longest Increasing Subsequence(c#)/LIS.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+using System ;
+class LIS
+{
+    /* lis() returns the length of the longest increasing
+    subsequence in arr[] of size n */
+    static int lis(int []arr,int n)
+    {
+        int []lis = new int[n];
+        int i,j,max = 0;
```
Comment (asha15) [inline Longest Increasing Subsequence(c#)/LIS.cs]: keep space after comma

File: Longest Increasing Subsequence(c#)/LIS.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                max = lis[i];
+
+            return max;
+    }
+
+    public static void Main()
+    {
+        int []arr = { 10, 22, 9, 33, 21, 50, 41, 60 };
+            int n = arr.Length;
+            Console.WriteLine("Length of lis is " + lis( arr, n ) + "\n" );
```
Comment (asha15) [inline Longest Increasing Subsequence(c#)/LIS.cs]: correct the indentation

File: Longest Increasing Subsequence(c#)/LIS.cs [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
```
Comment (asha15) [inline Longest Increasing Subsequence(c#)/LIS.cs]: remove this line

**Pull request closed without merging**